docs-client/src/components/VariableList/index.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+
+import { Specification } from '../../lib/specification';
```

Comment:
Global comment: The curly braces are redundant?

Reply:
I didn't add a "default export" to specification since it exports multiple things, and I've found in that case exporting a default can result in ugly looking imports like `import Specification, { Method, Service } from 'specification'`. I could add a default export though what do you think?

Reply:
I actually never knew `{ Something }` and `Something` are different. If they have different meaning, please leave them as they are.